iOS-tic-tac-toe/BoardStackView.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import UIKit
+
+public class BoardStackView: UIStackView, BoardView {
+    
+    public func show(board board: [String]) {
+        for view in self.subviews{
+            for button in view.subviews {
```

Comment:
Looping through all the `subviews` seems potentially fragile. I would look into if there is a way to wire the buttons as outlets. This might require a larger refactor, since I think you would need to have this be a `UIViewController` instead of just a `UIStackView`.

Let's plan on discussing this when I am back next week.